src/flatbuffers/fb_network_filter_generated.rs

```diff
@@ -255,7 +255,7 @@ pub mod fb {
         pub tag: Option<flatbuffers::WIPOffset<&'a str>>,
         pub raw_line: Option<flatbuffers::WIPOffset<&'a str>>,
     }
-    impl<'a> Default for NetworkFilterArgs<'a> {
+    impl Default for NetworkFilterArgs<'_> {
```

Comment:
we probably don't want to apply clippy to `_generated.rs` files?

Reply:
ops, missed that part. Fixed.